.golangci.yml

```diff
@@ -90,6 +90,17 @@ linters:
         - name: var-declaration
         - name: var-naming
 
+formatters:
+  enable:
+    - gci
```

Comment:
As stated in a recent discussion on golangci-lint, gci doesn't add or remove the missing imports, it simply sortw them so you have to use goimports in addition of gci

example

```go
package foo

import "fmt"

var _ = io.EOF
```

This file is broken

If you apply,  

```yaml
version: "2"

formatters:
  enable:
    - gci
    - gofmt
  settings:
    gci:
      sections:
        - standard
        - default
        - localmodule
```

It won't format, but it will if you add `goimports`

For me it's a bug